src/main.rs

```diff
@@ -139,27 +141,37 @@ fn main() -> anyhow::Result<()> {
                 notify::event::DataChange::Content,
             )) = event.kind
             {
-                let shape = match model.load(&parameters) {
-                    Ok(shape) => shape,
-                    Err(model::Error::Compile) => {
-                        // It would be better to display an error in the UI,
-                        // where the user can actually see it. Issue:
-                        // https://github.com/hannobraun/fornjot/issues/30
-                        println!("Error compiling model");
-                        return;
-                    }
-                    Err(err) => {
-                        panic!("Error reloading model: {:?}", err);
-                    }
-                };
-
-                // This will panic, if the other end is disconnected, which is
-                // probably the result of a panic on that thread, or the
-                // application is being shut down.
-                //
-                // Either way, not much we can do about it here, except maybe to
-                // provide a better error message in the future.
```

Comment:
You're removing this comment, even though it still applies in the changed version. Can you please re-add it?